### PR TITLE
Improved OSM log out text

### LINKED ISSUE #9046: [android] Not clear what does OSM Logout button do
It's hard to guess what's going on. A dialog text should be modified into something like "Are you sure you want to log out from your OSM account?"
выява

### PR DESCRIPTION
**Pull request by map-per**
fixes  #9046

Changed the logout text from "Are you sure you want to continue?" to "Are you sure you want to log out of your OpenStreetMap account?"

The `are_you_sure` string is not used anywhere else, so it can be changed safely.

| before | now |
|--------|--------|
|Screenshot_2024-08-22-17-05-43-286_app organicmaps beta|Screenshot_2024-08-22-16-59-20-472_app organicmaps debug|

### CODE REVIEW

**Comment by biodranik** [comment]
Thanks! It would be better to also change the id, as the current translation may be required in some other places later.

**Comment by map-per** [comment]
:+1: I will change it to `osm_log_out_confirmation`

**Comment by NeatNit** [comment]
In Hebrew I can't find a way to translate it gender-neutrally. The 'sure' is unavoidably gendered. The auto translation is fine.

**Comment by muralito** [comment]
> In Hebrew I can't find a way to translate it gender-neutrally. The 'sure' is unavoidably gendered. The auto translation is fine.

Same issue in spanish.

Some gender neutral option in spanish could be: "Quiere cerrar la sesion con la cuenta de OpenStreetMap"  ("Quiere" is more or less in english "Do you want to..."). Maybe that helps...

File: android/app/src/main/res/values-es/strings.xml
Comment (muralito) [inline android/app/src/main/res/values-es/strings.xml:1]: gender neutral, just; "¿Desea salir de su cuenta de OpenStreetMap?" or "¿Quiere salir de su cuenta de OpenStreetMap?"

File: data/strings/strings.txt [hunk 1]
```diff
@@ -4997,49 +4997,50 @@
...
+    ar = هل أنت متأكد أنك تريد تسجيل الخروج من حساب OpenStreetMap الخاص بك؟
+    az = OpenStreetMap hesabınızdan çıxmaq istədiyinizə əminsiniz?
+    be = Вы ўпэўнены, што хочаце выйсці са свайго ўліковага запісу OpenStreetMap?
+    bg = Сигурни ли сте, че искате да излезете от профила си в OpenStreetMap?
+    ca = Esteu segur que voleu tancar la sessió del vostre compte d'OpenStreetMap?
+    cs = Opravdu se chcete odhlásit ze svého účtu OpenStreetMap?
+    da = Er du sikker på, at du vil logge ud af din OpenStreetMap-konto?
+    de = Sind Sie sicher, dass Sie sich von Ihrem OpenStreetMap-Konto abmelden möchten?
+    el = Είστε σίγουροι ότι θέλετε να αποσυνδεθείτε από το λογαριασμό σας στο OpenStreetMap;
+    es = ¿Está seguro de que desea salir de su cuenta de OpenStreetMap?
```
Comment (muralito) [inline data/strings/strings.txt:5013]: Shorter and gender neutral alternatives "¿Quiere salir de su cuenta de OpenStreetMap?" or "¿Desea salir de su cuenta de OpenStreetMap?"

**Comment by strump** [comment]
Should we also add **"You local edits will not be shared with the OSM community!"** or simmilar notification to the message?

File: data/strings/strings.txt [hunk 2]
```diff
@@ -4997,49 +4997,50 @@
...
+    ja = 本当にOpenStreetMapアカウントからログアウトしますか？
+    ko = 오픈스트리트맵 계정에서 로그아웃하시겠습니까?
+    lt = Ar tikrai norite atsijungti nuo "OpenStreetMap" paskyros?
+    mr = तुमची खात्री आहे की तुम्ही तुमच्या OpenStreetMap खात्यातून लॉग आउट करू इच्छिता?
+    nb = Er du sikker på at du vil logge ut av OpenStreetMap-kontoen din?
+    nl = Weet u zeker dat u wilt uitloggen uit uw OpenStreetMap account?
+    pl = Czy na pewno chce się Pan/Pani wylogować ze swojego konta OpenStreetMap?
+    pt = Tem a certeza de que quer sair da sua conta OpenStreetMap?
+    pt-BR = Você tem certeza de que deseja sair da sua conta do OpenStreetMap?
+    ro = Sunteți sigur că doriți să vă deconectați din contul OpenStreetMap?
```
Comment (vgrigoriu) [inline data/strings/strings.txt:5033]: Gender-neutral alternative:
```suggestion
    ro = Sigur doriți să vă deconectați din contul OpenStreetMap?
```

**Comment by map-per** [comment]
> Should we also add "You local edits will not be shared with the OSM community!" or simmilar notification to the message?

I'm not sure, but I think adding that would be overdoing it. I think it's obvious, at least for most users, that uploading only works with an OSM account.

File: data/strings/strings.txt [hunk 3]
```diff
@@ -4997,49 +4997,50 @@
...
+    id = Apakah Anda yakin ingin keluar dari akun OpenStreetMap Anda?
+    it = È sicuro di voler uscire dal suo account OpenStreetMap?
+    ja = 本当にOpenStreetMapアカウントからログアウトしますか？
+    ko = 오픈스트리트맵 계정에서 로그아웃하시겠습니까?
+    lt = Ar tikrai norite atsijungti nuo "OpenStreetMap" paskyros?
+    mr = तुमची खात्री आहे की तुम्ही तुमच्या OpenStreetMap खात्यातून लॉग आउट करू इच्छिता?
+    nb = Er du sikker på at du vil logge ut av OpenStreetMap-kontoen din?
+    nl = Weet u zeker dat u wilt uitloggen uit uw OpenStreetMap account?
+    pl = Czy na pewno chce się Pan/Pani wylogować ze swojego konta OpenStreetMap?
+    pt = Tem a certeza de que quer sair da sua conta OpenStreetMap?
```
Comment (matheusgomesms) [inline data/strings/strings.txt:5031]:
```suggestion
    pt = Tem certeza de que deseja sair da sua conta do OpenStreetMap?
```

File: data/strings/strings.txt [hunk 4]
```diff
@@ -4997,49 +4997,50 @@
...
+    hu = Biztos, hogy ki akarsz jelentkezni az OpenStreetMap fiókodból?
+    id = Apakah Anda yakin ingin keluar dari akun OpenStreetMap Anda?
+    it = È sicuro di voler uscire dal suo account OpenStreetMap?
+    ja = 本当にOpenStreetMapアカウントからログアウトしますか？
+    ko = 오픈스트리트맵 계정에서 로그아웃하시겠습니까?
+    lt = Ar tikrai norite atsijungti nuo "OpenStreetMap" paskyros?
+    mr = तुमची खात्री आहे की तुम्ही तुमच्या OpenStreetMap खात्यातून लॉग आउट करू इच्छिता?
+    nb = Er du sikker på at du vil logge ut av OpenStreetMap-kontoen din?
+    nl = Weet u zeker dat u wilt uitloggen uit uw OpenStreetMap account?
+    pl = Czy na pewno chce się Pan/Pani wylogować ze swojego konta OpenStreetMap?
```
Comment (mpawelski) [inline data/strings/strings.txt:5030]: Definitely too formal, should be:
"Czy na pewno chcesz się wylogować ze swojego konta OpenStreetMap?"

File: data/strings/strings.txt [hunk 5]
```diff
@@ -4997,49 +4997,50 @@
...
+    el = Είστε σίγουροι ότι θέλετε να αποσυνδεθείτε από το λογαριασμό σας στο OpenStreetMap;
+    es = ¿Está seguro de que desea salir de su cuenta de OpenStreetMap?
+    et = Kas olete kindel, et soovite oma OpenStreetMapi kontost välja logida?
+    eu = Ziur zure OpenStreetMap kontutik saioa amaitu nahi duzula?
+    fa = آیا مطمئن هستید که می خواهید از حساب OpenStreetMap خود خارج شوید؟
+    fi = Haluatko varmasti kirjautua ulos OpenStreetMap-tililtäsi?
+    fr = Êtes-vous sûr de vouloir vous déconnecter de votre compte OpenStreetMap ?
+    he = האם אתה בטוח שברצונך להתנתק מחשבון OpenStreetMap שלך?
+    hi = क्या आप वाकई अपने OpenStreetMap खाते से लॉग आउट करना चाहते हैं?
+    hu = Biztos, hogy ki akarsz jelentkezni az OpenStreetMap fiókodból?
```
Comment (d4f5409d) [inline data/strings/strings.txt:5021]:
```suggestion
    hu = Biztos, hogy ki akar jelentkezni az OpenStreetMap fiókjából?
```

**Review by d4f5409d — CHANGES_REQUESTED** [review]
Hu needs a little change to apply formal language to make the app translation consistent.

**Comment by map-per** [comment]
Thank you for the reviews! I updated the translations and changed the id to `osm_log_out_confirmation`

**Pull request merged**